### PR TITLE
Using bigger node for minions for jenkin-e2e tests.

### PR DESCRIPTION
**Pull request by dchen1107**
cc/ @brendandburns @quinton-hoole 

This is not a fix for #9312, but it should help us to rule out failure such as:
https://github.com/GoogleCloudPlatform/kubernetes/issues/9312#issuecomment-110784030

### CODE REVIEW

**Comment by quinton-hoole** [comment]
LGTM

File: hack/jenkins/e2e.sh [hunk 1]
```diff
@@ -53,6 +54,7 @@ else
       export MASTER_SIZE=${MASTER_SIZE:-"t2.small"}
     else
       export MASTER_SIZE=${MASTER_SIZE:-"g1-small"}
```
Comment (brendandburns) [inline hack/jenkins/e2e.sh]: Change to n1-standard-1 too?

Comment (quinton-hoole) [inline hack/jenkins/e2e.sh]: Totally agree.  We should never be running on shared cores for CI e2e tests.  It introduces unnecessary timing variability, which makes debugging more difficult.

Comment (bprashanth) [inline hack/jenkins/e2e.sh]: Our performance goals are phrased in the context of an n1-standard-4 master (https://github.com/GoogleCloudPlatform/kubernetes/issues/6454)

**Comment by dchen1107** [comment]
Done!

**Pull request merged**